src/Dflydev/FigCookies/Modifier/SameSite.php

```diff
@@ -24,25 +26,25 @@ private function __construct(string $value)
         $this->value = $value;
     }
 
-    public static function strict() : self
+    public static function strict(): self
     {
-        return new self(self::STRICT);
+        return new static(self::STRICT);
```

Comment:
it's a final class, static is useless here

Reply:
But is it wrong?